spark/src/test/scala/filodb.spark/StreamingTest.scala

```diff
@@ -1,19 +1,19 @@
 package filodb.spark
 
 import com.typesafe.config.ConfigFactory
-import org.apache.spark.{SparkContext, SparkException, SparkConf}
+import org.apache.spark.{SparkConf, SparkContext, SparkException}
 import org.apache.spark.rdd.RDD
-import org.apache.spark.sql.{SaveMode, SQLContext}
+import org.apache.spark.sql.{SQLContext, SaveMode}
 import org.apache.spark.streaming.{Milliseconds, StreamingContext}
 import org.scalatest.time.{Millis, Seconds, Span}
+
 import scala.collection.mutable
 import scala.concurrent.duration._
-
-import filodb.core._
-import filodb.core.metadata.{Column, Dataset}
-import filodb.core.store.SegmentSpec
-
-import org.scalatest.{FunSpec, BeforeAndAfter, BeforeAndAfterAll, Matchers}
+import _root_.filodb.core._
```

Comment:
Please get rid of all these import _root_.filodb stuff, the _root_ is not necessary.